samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -152,7 +152,19 @@ class Cache(BaseModel):
     TransitEncryptionEnabled: Optional[PassThroughProp]
 
 
+class Defaults(BaseModel):
+    # properties common for Resolvers and Functions
+    Runtime: Optional[Runtime]
+    MaxBatchSize: Optional[PassThroughProp]
+    Sync: Optional[Sync]
+    # only Resolver properties
+    Resolvers: Optional[Resolver]
```

Comment:
nit: technically these aren't correct, as this means if `Resolvers` is provided every property required by `Resolver` must be specified. It's more of a "`Resolver` but all its properties are optional", similar to TypeScript's [`Partial`](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype), but not sure if there's anything similar

Reply:
That's a good catch. Though all props are optional now in Resolver, I thought at some point to have a separate type just in case. Well, it's a separate type in a new version anyway